docs/resources/dws_cluster_v1.md

```diff
@@ -50,61 +50,71 @@ resource "flexibleengine_dws_cluster_v1" "cluster" {
 
 The following arguments are supported:
 
-* `name` - (Required) Cluster name, which must be unique and contains 4 to 64
-    characters, which consist of letters, digits, hyphens (-), or underscores
-    (_) only and must start with a letter.
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the DWS cluster (v1) resource.
+  If omitted, the provider-level region will be used. Changing this will create a new DWS cluster (v1) resource.
 
-* `node_type` - (Required) Node type.
+* `name` - (Required, String, ForceNew) Cluster name, which must be unique and contains 4 to 64
+    characters, which consist of letters, digits, hyphens (-), or underscores(_) only and must start with a letter.
+    Changing this will create a new resource.
```

Comment:
```suggestion
* `name` - (Required, String, ForceNew) Cluster name, which must be unique and contains 4 to 64
  characters, which consist of letters, digits, hyphens (-), or underscores(_) only and must start with a letter.
  Changing this will create a new resource.
```